docs/docs/api/expansion/api-reference.md

```diff
@@ -12,10 +12,11 @@ The expansion service wraps the `route`, `isochrone` and `sources_to_targets` se
 
 Since this service wraps other services, the request format mostly follows the ones of the [route](../turn-by-turn/api-reference.md#inputs-of-a-route), [isochrone](../isochrone/api-reference.md#inputs-of-the-isochrone-service) and [matrix](../matrix/api-reference.md#inputs-of-the-matrix-service). Additionally, it accepts the following parameters:
 
-| Parameter                         | Description                           |
+| Parameter                         | Description                                                                                                                                                                                                                                                                                                                          |
```

Comment:
nit: probably dont need the whitespace

Reply:
Oh, sorry, I've overlooked this, it's fixed now